### PR TITLE
Cloud Events integration #128

### PR DESCRIPTION
**Pull request by rigazilla**
Enable Cloud event module in infinispan.xml.
fixes #infinispan-images#128

### CODE REVIEW

File: config-generator/src/main/resources/templates/infinispan.xml [hunk 1]
```diff
@@ -57,7 +60,9 @@
     <cache-container name="default" statistics="true" zero-capacity-node="{infinispan.zeroCapacityNode}">
         <transport cluster="$\{infinispan.cluster.name:{infinispan.clusterName}\}" node-name="$\{infinispan.node.name:\}"
         {#if xsite.backups}stack="xsite"{#else}stack="image-{jgroups.transport}"{/if}/>
-
+        {#if cloudEvents}
+            <ce:cloudevents {#if cloudEvents.boostrapServer}bootstrap-servers="{cloudEvents.bootstrapServer}"{/if} {#if cloudEvents.acks}acks="{cloudEvents.acks}"{/if} {#if cloudEvents.cacheEntriesTopic}cache-entries-topic="{cloudEvents.cacheEntriesTopic}"{/if}/>
```
Comment (ryanemerson) [inline config-generator/src/main/resources/templates/infinispan.xml:64]: Isn't `bootstrap-servers` mandatory? In which case we can just do `bootstrap-servers="{cloudEvents.bootstrapServer}"`


This also applies to acks and cache-entries-topic if they are mandatory, but I'm not sure if that's the case?

**Review by ryanemerson — COMMENTED** [review]
It would be good to add a test case to `AbstractMainTest` just to make sure that nothing gets broken with future template changes.

File: config-generator/src/main/resources/templates/infinispan.xml (hunk 1 above)
Comment (rigazilla) [reply config-generator/src/main/resources/templates/infinispan.xml:64]: uh yeah `bootstrap-servers` is mandatory, `acks` and `cache-entries-topic` have defaults value if not provided.

**Comment by ryanemerson** [comment]
Thanks @rigazilla

**Pull request merged**